kernel/build/config-amd64

```diff
@@ -2083,6 +2095,9 @@ CONFIG_SYSFB=y
 # EFI (Extensible Firmware Interface) Support
 #
 CONFIG_EFI_ESRT=y
+CONFIG_EFI_VARS_PSTORE=y
```

Comment:
Is this disabled by default? Polluting UEFI NVRAM on crashes by default might not be desirable

Reply:
good point, I think we should disable it

Reply:
removed